x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/transform/action/GetTransformActionResponseTests.java

```diff
@@ -25,6 +25,15 @@
 
 public class GetTransformActionResponseTests extends AbstractWireSerializingTransformTestCase<Response> {
 
+    public static Response randomTransformResponse() {
+        List<TransformConfig> configs = new ArrayList<>();
+        for (int i = 0; i < randomInt(10); ++i) {
```

Comment:
This is generating a different random int for the stopping point on every iteration.  It will mean that getting to the high numbers like 8, 9, 10 is extremely rare.  It would be easier to understand if `int numConfigs = randomInt(10);` was pulled out of the loop condition and then `numConfigs` used in the loop condition.

Reply:
:+1: good spot, I just moved the code, nevertheless will fix this and other occurences